lib/simulation/openfisca/index.js

```diff
@@ -40,10 +40,15 @@ function calculate(situation, debug, callback) {
 }
 
 function simulate(situation, callback) {
+    var injectedRessources = _.unique(_.flatten(_.map(situation.individus, function(individu) {
+        return _.uniq(_.map(individu.ressources, function(ressource) {
```

Comment:
No need to `unique` twice. Also, be consistent between `uniq` and `unique` synonyms.